extensions/applicationinsights-properties-js/src/Context/Session.ts

```diff
@@ -166,9 +169,9 @@ export class _SessionManager {
     }
 
     private renew() {
-        const now = DateTimeUtils.Now();
+        const now = new Date().getTime();
```

Comment:
As above, avoid performance API bugs and we don't need the additional floating point precision.